fortdepend/units.py

```diff
@@ -46,8 +46,8 @@ def __init__(
         self.depends_on = None
 
         if readfile:
-            with smart_open(self.filename, "r") as f:
-                contents = f.read()
+            with smart_open(self.filename, "rb") as f:
+                contents = f.read().decode(errors="ignore").lower()
```

Comment:
Could this just be:

```suggestion
            with smart_open(self.filename, "r", encoding="utf-8") as f:
                contents = f.read().lower()
```

Ah, is this needed in order to have `errors="ignore"`?